packages/odyssey-react/src/utils/focusHandling.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+type OptionalHTMLElement = HTMLElement | null;
+
+interface UseFocusHook {
+  restoreFocus: (current: OptionalHTMLElement) => void;
+  setFocus: (elem: OptionalHTMLElement) => OptionalHTMLElement;
+}
+
+const FOCUSABLE_ITEMS = [
+  "button",
+  "[href]",
+  "input",
+  "select",
+  "textarea",
+  '[tabindex]:not([tabindex="-1"])',
```

Comment:
this won't work for IE11 I believe. We'll need to filter our tabindex values less than 0 using imperative code here